tests/conftest.py

```diff
@@ -1,3 +1,14 @@
 # Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
-#
+
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Generator
+
+import pytest
+
+
+@pytest.fixture(scope="session")
+def tmp_dir_path() -> Generator[Path, None, None]:
+    with TemporaryDirectory() as tmp_dir:
+        yield Path(tmp_dir)
```

Comment:
You can use the best practice https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture

Reply:
I didn't realize this, thanks for pointing it out, I've fixed all the tmp_dir_paths to tmp_path.
https://github.com/openvinotoolkit/training_extensions/commit/39c65454f5bd5aeff26fbb60925e0f1064b993bb